requirements-common.txt

```diff
@@ -19,3 +19,4 @@ lm-format-enforcer == 0.10.1
 outlines == 0.0.34 # Requires torch >= 2.1.0
 typing_extensions
 filelock >= 3.10.4 # filelock starts to support `mode` argument from 3.10.4
+bitsandbytes == 0.42.0
```

Comment:
This doesn't belong in the common requirements and actually I think we shouldn't require bitsandbytes by default in the package. I think in `vllm/model_executor/layers/quantization/bitsandbytes.py` we should check if it is installed and throw then. This is what was done for `deepspeedfp` https://github.com/vllm-project/vllm/blob/606625329648e6eff1883e23040adfad82f219cf/vllm/model_executor/layers/quantization/deepspeedfp.py#L145-L154

Reply:
fixed.